crates/bevy_ecs/src/world/mod.rs

```diff
@@ -264,7 +271,13 @@ impl World {
     /// ```
     #[inline]
     pub fn get_entity_mut(&mut self, entity: Entity) -> Option<EntityMut> {
-        let location = self.entities.get(entity)?;
+        let location = match self.entities.get(entity) {
+            Ok(entity_location) => entity_location,
+            Err(err) => {
+                warn!("Error getting entity {:?}: {:?}", entity, err,);
```

Comment:
```suggestion
                warn!("Error getting entity {:?}: {:?}", entity, err);
```

I think this trailing comma is here by mistake?